lib/pubnub/events/revoke_token.rb

```diff
@@ -0,0 +1,66 @@
+module Pubnub
+  # Holds revoke token functionality
+  class RevokeToken < SingleEvent
+    include Concurrent::Async
+    include Pubnub::Validator::RevokeToken
+
+    def initialize(options, app)
+      @event = :revoke_token
+      super
+      @token = @token.split(' ')
+            .map{ |part| CGI.escape(part) }
+            .join("%20")
```

Comment:
I'm doing it because we have a test scenario in contract tests that contains space in the token. I wanted to use `URI.escape` but it's apparently deprecated and it suggest using `CGI.escape`. If you think it's ok to use `URI.escape` and it'll do what it suppose to do I can use it

Reply:
@parfeon I checked the `URI.escape` and it's not working as I would like

Reply:
@kleewho ok.